### ISSUE TITLE
Livn | Experience Details page

### ISSUE THREAD

**Issue by Justin-Lag** [Guestlogix/traveler-android#192]
Child of guestlogix/traveler-android#191
blocked by guestlogix/traveler-android#198

As a traveler I want select what variation of the experience I want to book so that I can have an experience that meets my travel goals

**Acceptance Criteria**
1. Each "Tour" may be available in many different "Flavours".   The traveler must select an available "flavour" to book an experience
1. If "flavours" aren't offered, then the flavour selection control isn't visible

**Out of scope/Future consideration**
Any relevant out of scope items

**UX if needed**
No massive UI changes here for now, using drop-down to display the different "options"
**Design:**https://zpl.io/aR1dE7E
*Tag:* LIVN

https://www.notion.so/Livn-Buy-Flow-7f7c6be982804354a2437ce90db2911f

**Technical details if needed**
[DB]
[API]

**Testing considerations**
